data/transform/models/marts/telemetry/base/prep_project_lifetime.sql

```diff
@@ -0,0 +1,69 @@
+{{
+    config(
+        materialized='table',
+        description="A prep table to calculate relative project dates, ages, etc."
+    )
+}}
+
+WITH cte_max_date_thresholds AS (
+    # Generates a single row containing the max date (aka "today" based on upsteam EL)
+    SELECT
+        MAX(cli_executions_base.event_created_at)::DATE AS latest_data_date,
+        MAX(cli_executions_base.event_created_at)       AS latest_data_timestamp,
+    FROM {{ ref('cli_executions_base') }} AS cli_executions_base
+),
+
+cte_vital_dates AS (
```

Comment:
```suggestion
cte_project_dates AS (
```

@aaronsteers should this be updated? It looks like its referenced as cte_project_dates later?